ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -49,53 +51,61 @@ struct MSFListCellView: View {
             }
         }, label: {
             HStack(spacing: 0) {
+                let defaultIconSize = tokens.iconDefaultSize
+                let hasTitle = !state.title.isEmpty
                 if let leadingView = state.leadingView {
                     UIViewAdapter(leadingView)
-                        .frame(width: tokens.iconSize, height: tokens.iconSize)
+                        .foregroundColor(Color(hasTitle ? tokens.backgroundColor : tokens.trailingItemForegroundColor))
+                        .frame(width: state.isLargeIcon ? tokens.iconLargeSize : defaultIconSize, height: state.isLargeIcon ? tokens.iconLargeSize : defaultIconSize)
                         .padding(.trailing, tokens.iconInterspace)
                 }
-                VStack(alignment: .leading) {
-                    if let title = state.title {
-                        Text(title)
+                VStack(alignment: .leading, spacing: 0) {
+                    if hasTitle {
+                        Text(state.title)
                             .font(Font(tokens.textFont))
                             .foregroundColor(Color(tokens.leadingTextColor))
-                            .lineLimit(state.titleLineLimit)
+                            .lineLimit(state.titleLineLimit == 0 ? nil : state.titleLineLimit)
```

Comment:
over reliance on state blocks the functionality or interfaces for native swiftUI clients, how would a swiftUI client consume or set the line limit?

Reply:
If the client chooses to set the line limit then they would set it in the state along with the rest of the cell properties. By default there is no line limit set.